### ISSUE TITLE
Try to get rid of compiling warnings

### ISSUE THREAD

**Issue by bingen** [aragon/aragonOS#385]
Main rationale:
https://github.com/ethereum/solidity/issues/4315#issuecomment-398466651
Besides, it would make easier to spot compiling errors while developing, that sometimes get buried among long list of warnings.

**Comment by izqui** [comment]
#383 removes almost all the compiler warnings in our code. I actually listed the warnings that are still standing.

**Comment by bingen** [comment]
So should we close this one? Do you want to put a "Closing .." footer to #383 so this one gets closed once that one is merged?

**Comment by izqui** [comment]
Yep, we should close this with #383 and then create issues for the remaining warnings so we can track them separately.
